internal/store/bigtable/group.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bigtable
+
+import (
+	"context"
+	"sort"
+	"strings"
+	"sync"
+
+	cbt "cloud.google.com/go/bigtable"
+)
+
+var groupRank = map[string]int{
+	"frequent":   0,
+	"dc":         1,
```

Comment:
dc_branch ?

Reply:
The split is by "_" so the first seen element is "dc".. Would be nice if the name is dcbranch as one word.

Reply:
We can make the change when removing "borgcron_".